src/screens/CategoriesFilterScreen/Header.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+import React from "react";
+import { View, TouchableOpacity, StyleSheet } from "react-native";
+import Text from "../../components/Text";
+import text from "../../constants/text";
+import { whiteColor } from "../../constants/colors";
+import CategoriesPills from "../../components/CategoriesPills";
+
+type Props = {
+  onClose?: Function,
+  onClearAll?: Function,
+  selectedCategories: Set<string>
+};
+
+type ActionButtonProps = {
+  label: string,
+  onPress?: Function
+};
+
+const ActionButton = ({ label, onPress }: ActionButtonProps) => (
+  <TouchableOpacity style={styles.actionButton} onPress={onPress}>
```

Comment:
Can you use our `Touchable` component here to get the native styling on Android and the accessibility defaults?